domain/infosync/info.go

```diff
@@ -405,6 +406,8 @@ func doRequest(ctx context.Context, addrs []string, route, method string, body i
 			}
 			terror.Log(res.Body.Close())
 			return bodyBytes, err
+		} else {
+			logutil.BgLogger().Error("fail to doRequest", zap.Error(err), zap.String("server addr", addr))
```

Comment:
1. level Warn is better? 
2. message to "fail to doRequest, retry next address" 
3. I think we can add more information like addressIndex, method and route.

Reply:
Done